OpenOversight/app/utils.py

```diff
@@ -610,8 +610,10 @@ def prompt_yes_no(prompt, default="no"):
             continue
         return ret
 
-    
+
 def normalize_gender(input_gender):
+    if input_gender is None:
+        return None
```

Comment:
This is also needed for the csv-import functionality and tests of it

Reply:
do you want me to add something in the original PR to account for this?

Reply:
It think that's the only place this is needed. Not sure why I wrote "also"